### PR TITLE
Dw 208 remove animations and improve performance

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Add link to index
- image compression for better performance
- typography removal in titles for better performance
- removal of animations to improve performance

new-layout

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Comment by GusBaamonde** [comment]
performance-state

**Comment by GusBaamonde** [comment]
performance-state2

**Comment by GusBaamonde** [comment]
performance-audits

**Comment by GusBaamonde** [comment]
test this please

File: assets/templates/account-message-component.html [hunk 1]
```diff
@@ -3,18 +3,24 @@
...
+    />
+    <meta
+      name="description"
+      content="These messages appear above the header as account statements of our users."
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="canonical" href="" />
     <!-- Open Graph metatags -->
-    <meta property="og:title" content="" />
+    <meta property="og:title" content="Account Message component" />
```
Comment (cbernat) [inline assets/templates/account-message-component.html]: What does it means `account message component`? 🤔

Comment (cbernat) [reply assets/templates/account-message-component.html]: Forget about this I get the meaning now 😄

**Comment by cbernat** [comment]
Very nice! 💪

**Pull request merged**